docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -3,31 +3,42 @@
 
 **Tip**: If you are new to AutoGluon, review :ref:`sec_tabularquick` to learn the basics of the AutoGluon API.
 
-This tutorial describes how you can exert greater control when using AutoGluon's `fit()` by specifying the appropriate arguments. Using the same census data table as :ref:`sec_tabularquick`, we will try to predict the `occupation` of an individual - a multi-class classification problem.
+This tutorial describes how you can exert greater control when using AutoGluon's `fit()` or `predict()`. Recall that to maximize predictive performance, you should always first try `fit()` with all default arguments except `eval_metric` and `presets`, before you experiment with other arguments covered in this in-depth tutorial like `hyperparameter_tune`, `hyperparameters`, `stack_ensemble_levels`, `num_bagging_folds`, `num_bagging_sets`, etc.
 
-Start by importing AutoGluon, specifying TabularPrediction as the task, and loading the data.
+Using the same census data table as before, we'll now predict the `occupation` of an individual - a multiclass classification problem. Start by importing AutoGluon, specifying TabularPrediction as the task, and loading the data.
 
 ```{.python .input}
 import autogluon as ag
 from autogluon import TabularPrediction as task
 
+import numpy as np
+
 train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
-train_data = train_data.head(500) # subsample 500 data points for faster demo (comment this out to run on full dataset instead)
+subsample_size = 500 # subsample subset of data for faster demo, try setting this to much larger values
+train_data = train_data.sample(n=subsample_size, random_state=0)
 print(train_data.head())
 
-val_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
-
 label_column = 'occupation'
 print("Summary of occupation column: \n", train_data['occupation'].describe())
+
+val_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
```

Comment:
maybe call this unseen_data / new_data? Having it be called val_data and then test_data is being derived from it seems a bit confusing for novices. val_data can still be specified as `val_data = unseen_data[:5000]`